src/core/js/accessibility.js

```diff
@@ -59,41 +48,38 @@ define([
             this.listenTo(Adapt, 'pageView:ready menuView:ready router:plugin', this.onNavigationEnd);
         },
 
+        removeLegacyElements: function() {
+            var $legacyElements = $("body > #accessibility-toggle, body > #accessibility-instructions");
```

Comment:
Would you mind switching to the more concise and efficient `$.children()`?
```js
$("body").children("#accessibility-toggle, #accessibility-instructions");
```
And switching the logic to return early?